src/App.css

```diff
@@ -61,6 +61,44 @@
    vertical-align: middle;
 }
 
+.delete-button {
+  margin-left: 10px;
+  width:  48px;
+  height: 48px; 
+  display: flex;
+  border-radius: 10px;
+}
+
+.new-task-container{
+  margin-top: 50px;
+  display: flex;
+  width: 770px;
+  height: 67px;
+  text-align: left;
+}
+
+
+.newtask-name {
+  width: 730px;
+  height: 48px;
+  text-align: left;
+  border-radius: 10px;
+  display: flex;
+}
+
+.newtask-button {
+  margin-left: 10px;
+  padding: 15px 0px 0px 25px;
+  display: flex;
+  width:  107.97px;
```

Comment:
There shouldn't be a static `width` for this, it should stretched based on the remaining space left